pydantic_ai_slim/pyproject.toml

```diff
@@ -75,7 +75,7 @@ tavily = ["tavily-python>=0.5.0"]
 # CLI
 cli = ["rich>=13", "prompt-toolkit>=3", "argcomplete>=3.5.0"]
 # MCP
-mcp = ["mcp>=1.6.0; python_version >= '3.10'"]
+mcp = ["mcp>=1.8.0; python_version >= '3.10'"]
```

Comment:
note that I think this may need to be 1.9.0 as the protocol version in 1.8.0 is still the 2024-11-05 one. we want the 2025-03-26 one (latest) as that's where the streamable HTTP protocol is defined

Reply:
https://github.com/modelcontextprotocol/python-sdk/releases/tag/v1.9.0